packages/CodeGenLib/src/Database/manage-metadata.ts

```diff
@@ -1142,6 +1256,7 @@ export class ManageMetadataBase {
                if (!possibleValues.find(v => v === ev.Value)) {
                   // delete the value from the database
                   const sSQLDelete = `DELETE FROM [${mj_core_schema()}].EntityFieldValue WHERE ID='${ev.ID}'`;
+                  this.appendToFile(sSQLDelete, `SQL text to delete entity field value ID ${ev.ID}`);
```

Comment:
@JS-BC  the reason to use a wrapper? well, that way we can modify the behavior in one place in the future if we want to do something else, make sense?

Reply:
Makes sense